ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/SPGDPRManagePreferenceViewController.swift

```diff
@@ -106,16 +109,16 @@ class SPGDPRManagePreferenceViewController: SPNativeScreenViewController {
 extension SPGDPRManagePreferenceViewController: UITableViewDataSource, UITableViewDelegate {
     func currentCategory(_ index: IndexPath) -> GDPRCategory {
         displayingLegIntCategories ?
-            legIntCategories[index.row] :
-            sections[index.section].content[index.row]
+            sections[index.section].contentLegIntCategory![index.row] :
```

Comment:
@Nevazhnovu is it guaranteed that `contentLegIntCategory` is not `nil`? I'm asking because of that forced unwrapping.